Document missing activator-capacity (#3819)
knative/docs: docs/serving/load-balancing/activator-capacity.md

---
title: "Configuring Activator capacity"
linkTitle: "Configuring Activator capacity"
weight: 50
type: "docs"
---

# Configuring Activator capacity

> If there's more than one Activator in the system, Knative will put as many of them on the request path as it thinks is necessary (if target burst capacity is not set to 0). If available, it'll pick at least two for high availability reasons. The actual number of Activators is calculated via a given _Activator capacity_ like such: `(replicas * target + targetBurstCapacity)/activatorCapacity`. That means, there are enough Activators in the routing path to handle the theoretic capacity of the existing application, including any additional target burst capacity.

Reviewer: How about something (for the first sentence) like: "..Knative will put as many of them on to the request path as it thinks is necessary to handle the current request load plus the Target Burst Capacity. If Target Burst Capacity is 0 Knative only puts the activator in to the path when the revision is scaled to zero"

Reviewer: Also because I happen to know the style guide will not let us use parentheticals :D

---
title: "Configuring Activator capacity"
linkTitle: "Configuring Activator capacity"
weight: 50
type: "docs"
---

> 

Reviewer 2: This frontmatter doesn't need to be added anymore since we moved to mkdocs

# Configuring Activator capacity

> If there's more than one Activator in the system, Knative will put as many of them on the request path as it thinks is necessary to handle the current request load plus the target burst capacity. If the target burst capacity is 0, Knative only puts the Activator into the request path if the Revision is scaled to zero.

Reviewer 2: ```suggestion
If there is more than one Activator in the system, Knative puts as many Activators on the request path as required to handle the current request load plus the target burst capacity. If the target burst capacity is 0, Knative only puts the Activator into the request path if the Revision is scaled to zero.
```

> If there's more than one Activator in the system, Knative will put as many of them on the request path as it thinks is necessary to handle the current request load plus the target burst capacity. If the target burst capacity is 0, Knative only puts the Activator into the into the request path if the Revision is scaled to zero.

Reviewer: ```suggestion
If there's more than one Activator in the system, Knative will put as many of them on the request path as it thinks is necessary to handle the current request load plus the target burst capacity. If the target burst capacity is 0, Knative only puts the Activator into the request path if the Revision is scaled to zero.
```

> If there's more than one Activator in the system, Knative will put as many of them on the request path as it thinks is necessary to handle the current request load plus the target burst capacity. If the target burst capacity is 0, Knative only puts the Activator into the into the request path if the Revision is scaled to zero. 

Reviewer: ```suggestion
If there's more than one Activator in the system, Knative will put as many of them on the request path as it thinks is necessary to handle the current request load plus the target burst capacity. If the target burst capacity is 0, Knative only puts the Activator into the request path if the Revision is scaled to zero. 
```

> If available, Knative will pick at least two Activators for high availability reasons. The actual number of Activators is calculated via a given _Activator capacity_ like such: `(replicas * target + targetBurstCapacity)/activatorCapacity`. That means, there are enough Activators in the routing path to handle the theoretic capacity of the existing application, including any additional target burst capacity.

Reviewer: ```suggestion
If available, Knative will pick at least two Activators for high availability reasons. The actual number of Activators is calculated via a given _Activator capacity_ using the formula: `(replicas * target + targetBurstCapacity)/activatorCapacity`. That means, there are enough Activators in the routing path to handle the theoretic capacity of the existing application, including any additional target burst capacity.
```

> If available, Knative will pick at least two Activators for high availability reasons. The actual number of Activators is calculated via a given _Activator capacity_ using the formula: `(replicas * target + targetBurstCapacity)/activatorCapacity`. That means, there are enough Activators in the routing path to handle the theoretical capacity of the existing application, including any additional target burst capacity.

Reviewer 2: ```suggestion
Knative uses at least two Activators to enable high availability if possible. The actual number of Activators is calculated as the _Activator capacity_, by using the formula `(replicas * target + targetBurstCapacity)/activatorCapacity`. This means that there are enough Activators in the routing path to handle the theoretical capacity of the existing application, including any additional target burst capacity.
```

Author: This is not quite accurate. The actual number is not the Activator capacity, but the Activator capacity is a factor in the formula.

Reviewer 2: Updated it slightly

Author: I've already changed the wording a bit, please check if that's fine too.
